sql/core/src/test/scala/org/apache/spark/sql/execution/python/PythonDataSourceSuite.scala

```diff
@@ -161,12 +161,12 @@ class PythonDataSourceSuite extends QueryTest with SharedSparkSession {
       s"""
          |from pyspark.sql.datasource import DataSource, DataSourceReader
          |class SimpleDataSourceReader(DataSourceReader):
-         |    def __init__(self, paths, options):
```

Comment:
regardless we should remove this `paths`. Not all Python Datasources require `pathes`